client/galaxy/scripts/mvc/tours.js

```diff
@@ -10,7 +10,7 @@
 
 define(['libs/bootstrap-tour'],function(BootstrapTour) {
 
-    var gxy_root = Galaxy === undefined ? '/' : Galaxy.root;
+    var gxy_root = typeof Galaxy === "undefined" ? '/' : Galaxy.root;
```

Comment:
Regarding same check as the other backbone locations -- we assume a 'naked' Galaxy in the context in many other places -- this is actually an attempt to set a standard.  There isn't a good reason Galaxy has to be attached to window, it just has to exist in the parent scope.  

The question is "why assume window, when there's no need to?  we just need to check if there is a Galaxy handle in the parent (whatever that may be) context.".  It's a bit down the road, but we may have a webapp that interacts with multiple Galaxy instances at some point, etc, and 'window' will no longer work.  Libraries that don't assume a 'window' context are also one step closer to being reusable in other contexts (serverside nodejs, etc).